### ISSUE TITLE
Nice models

### ISSUE THREAD

**Issue by HappyDay-design** [kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit#1]
Hello
Thank you very much for sharing these materials. 
I would like to ask you about path results.
In `DWL_ST_VSL_2Agents.py `you set `path_results = "C:/Users/kusic/Desktop/DWL-ST-VSL/Results/"`

but in `plot_results.py` you set path_results = 
`"C:/Users/kusic/Desktop/KK/SUMO/WL_analysis/reward_freeSpeed_or_TTS/"+""+\"DWL_v2p1p2_betaPonder_1TTS3/GIT/MD_C_0_75_W_on_LP_RP/Results/"`

Is there any difference between them?

**Comment by kkusic** [comment]
Hello,

Thanks. If needed some help or an explanation, feel free to contact me.


Best
Kresimir

________________________________
Šalje: TrinhTuanHung2021 ***@***.***>
Poslano: 25. kolovoza 2022. 16:02
Prima: kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit ***@***.***>
Kopija: Subscribed ***@***.***>
Predmet: [kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit] Nice models (Issue #1)


Hello
Thank you very much for sharing these materials.

**Comment by kkusic** [comment]
Yes, it should be the same as the path you saved the results with the main script.
In my case, "C:/Users/kusic/Desktop/DWL-ST-VSL/Results/"

**Comment by HappyDay-design** [comment]
Hello. After training your model and plotting the results, it only shows the time-space diagram
![image](https://user-images.githubusercontent.com/82990411/194788773-6293606f-6ac1-4ef5-8a94-8019404734bc.png)

I would like to ask you how to get the training process like this.
![image](https://user-images.githubusercontent.com/82990411/194788835-3e3f0e7e-82fc-4f94-8bac-5fa35d87c913.png)

And in the last paper, you used 4 agents.
But in this model, you only used 2 agents (A1 and A2). 
Thank you very much!

**Comment by kkusic** [comment]
Hello,

this script is missing. As soon as I find time, I will add this script to repository.


Thanks
Kresimir

________________________________
Šalje: TrinhTuanHung2021 ***@***.***>
Poslano: 10. listopada 2022. 2:28
Prima: kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit ***@***.***>
Kopija: Krešimir Kušić ***@***.***>; Comment ***@***.***>
Predmet: Re: [kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit] Nice models (Issue #1)


Hello. After training your model and plotting the results, it only shows the time-space diagram
[image]<https://user-images.githubusercontent.com/82990411/194788773-6293606f-6ac1-4ef5-8a94-8019404734bc.png>

I would like to ask you how to get the training process like this.
[image]<https://user-images.githubusercontent.com/82990411/194788835-3e3f0e7e-82fc-4f94-8bac-5fa35d87c913.png>
Thank you very much!

**Comment by kkusic** [comment]
Hi,

I uploaded the script plot_TTS_convergence.py to the repository. Please run it and give feedback.
(just adjust paths to folder where you have stored results, it includes a curve for DWL-2 agents and NO control scenario)


Best
Kresimir

________________________________
Šalje: Krešimir Kušić ***@***.***>
Poslano: 10. listopada 2022. 8:02
Prima: kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit ***@***.***>
Predmet: Odg: [kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit] Nice models (Issue #1)

Hello,

this script is missing. As soon as I find time, I will add this script to repository.


Thanks
Kresimir

________________________________
Šalje: TrinhTuanHung2021 ***@***.***>
Poslano: 10. listopada 2022. 2:28
Prima: kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit ***@***.***>
Kopija: Krešimir Kušić ***@***.***>; Comment ***@***.***>
Predmet: Re: [kkusic/Multi_Agent_Based_Spatio_Temporal_Variable_Speed_Limit] Nice models (Issue #1)


Hello. After training your model and plotting the results, it only shows the time-space diagram
[image]<https://user-images.githubusercontent.com/82990411/194788773-6293606f-6ac1-4ef5-8a94-8019404734bc.png>

I would like to ask you how to get the training process like this.
[image]<https://user-images.githubusercontent.com/82990411/194788835-3e3f0e7e-82fc-4f94-8bac-5fa35d87c913.png>
Thank you very much!

**Comment by kkusic** [comment]
Regarding, DWL 4 agents and (WL and SPSC) I need more time to make the code readable. As soon as I fit them, I will upload them.

**Comment by HappyDay-design** [comment]
> Regarding, DWL 4 agents and (WL and SPSC) I need more time to make the code readable. As soon as I fit them, I will upload them.

Thank you very much.
